libraries/azure-app-configuration-importer-file-source/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/app-configuration-importer-file-source",
   "author": "Microsoft Corporation",
   "description": "A client library for importing/exporting key-values between configuration file sources and Azure App Configuration service",
-  "version": "1.0.1-preview",
```

Comment:
For the importer-file-source please help verify when its built its referencing version 1.1.0 of the importer library